webapp/components/wpt-results.html

```diff
@@ -507,11 +490,10 @@ <h3>
         }
 
         const result = node.results[index];
-        if (typeof result === 'undefined') {
+        const isDiff = this.isDiff(testRun);
+        if (!isDiff && typeof result === 'undefined') {
           return 'none';
```

Comment:
Should this also check that `prop === 'total'`? It's not really none unless there are no tests (no passes doesn't matter).

Reply:
Let's discuss tomorrow. I'm not quite sure about this.